sourcetools/buildj9tools.mk

```diff
@@ -46,7 +49,7 @@ ifdef DEV_TOOLS
   JAVA_BIN  ?= $(DEV_TOOLS)/ibm-jdk-1.8.0/bin
 else
   # requires Java 8 or 9
-  JAVA      := java8
+  JAVA      := java$(PREV_JAVA)
```

Comment:
Makes the asumptions that javaN is available on all machines that utilize run this code which may or may not be the case.